src/NUnitTestAdapter/TestKey.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ***********************************************************************
+// Copyright (c) 2018 Charlie Poole, Terje Sandstrom
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+namespace NUnit.VisualStudio.TestAdapter
+{
+    public struct TestKey
+    {
+        public readonly string FlowId;
```

Comment:
`FlowId` is confusing. If this simply holds the `id` and `name` attributes from the test XML, then this should be named `Id`.

Reply:
So per my [other comment](https://github.com/nunit/nunit3-vs-adapter/pull/544/files#r224937294), the `id` attribute string itself is the test key.
If you ever see anything that contradicts this, it's a critical bug in NUnit.
Therefore, the `TestKey` struct should not define equality to include the name parameter. To keep the PR simple, it should just be replaced with `string`.

Reply:
TestKey was removed